pkg/components/util/install.go

```diff
@@ -44,6 +44,9 @@ func ensureNamespaceExists(name string, kubeconfigPath string) error {
 	_, err = cs.CoreV1().Namespaces().Create(context.TODO(), &v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
+			Labels: map[string]string{
+				"lokomotive.kinvolk.io/name": name,
+			},
```

Comment:
This is unnecessary since after calling `ensureNamespaceExists` we immediately call `UpdateNamespaceWithLabels` which applies labels regardless.